docs/libcurl/curl_easy_init.3

```diff
@@ -58,8 +58,16 @@ if(curl) {
 .SH AVAILABILITY
 Always
 .SH RETURN VALUE
-If this function returns NULL, something went wrong and you cannot use the
-other curl functions.
+Returns a valid non-null handle on success.
```

Comment:
Maybe "it returns a pointer to a CURL easy handle on success." instead?

Talking about "valid non-null" might imply that there can be a invalid non-null somehow.